koap/build.gradle.kts

```diff
@@ -28,6 +27,12 @@ kotlin {
             }
         }
 
+        val jsMain by getting {
+            dependencies {
+                implementation(npm("os-browserify", "0.3.0"))
+            }
+        }
+
```

Comment:
okio multiplatform for JavaScript uses the OS module. Since we target the browser rather than node we have to provide the polyfill module